world_universities_and_domains.json

```diff
@@ -131221,5 +131223,848 @@
+      "http://www.nxmu.edu.cn/"
+    ],
+    "name": "Ningxia Medical University",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "nxmu.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.pace.edu.in"
+    ],
+    "name": "PA College of Engineering",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "pace.edu.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.sakec.ac.in"
+    ],
+    "name": "Shah and Anchor Kutchhi Engineering College",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "sakec.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.dituniversity.edu.in/"
+    ],
+    "name": "DIT University",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "dit.edu.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.cukerala.ac.in/"
+    ],
+    "name": "Central University of Kerala",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "cukerala.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.psit.ac.in/"
+    ],
+    "name": "Pranveer Singh Institute of Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "psit.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://website.robcol.k12.tr"
+    ],
+    "name": "Robert College",
+    "alpha_two_code": "TR",
+    "state-province": null,
+    "domains": [
+      "probcol.k12.tr"
+    ],
+    "country": "Turkey"
+  },
+  {
+    "web_pages": [
+      "https://www.vspj.cz"
+    ],
+    "name": "Vysoká škola polytechnická Jihlava",
+    "alpha_two_code": "CZ",
+    "state-province": null,
+    "domains": [
+      "student.vspj.cz"
+    ],
+    "country": "Czech Republic"
+  },
+  {
+    "web_pages": [
+      "https://www.udc.edu.br"
+    ],
+    "name": "UDC Centro Universitário",
+    "alpha_two_code": "BR",
+    "state-province": null,
+    "domains": [
+      "udc.edu.br"
+    ],
+    "country": "Brazil"
+  },
+  {
+    "web_pages": [
+      "https://www.rshu.edu.ua/"
+    ],
+    "name": "Rivne State University of the Humanities",
+    "alpha_two_code": "UA",
+    "state-province": null,
+    "domains": [
+      "rshu.edu.ua"
+    ],
+    "country": "Ukraine"
+  },
+  {
+    "web_pages": [
+      "https://www.yu.ac.kr"
+    ],
+    "name": "Yeungnam University",
+    "alpha_two_code": "KR",
+    "state-province": null,
+    "domains": [
+      "ynu.kr"
+    ],
+    "country": "Korea"
+  },
+  {
+    "web_pages": [
+      "https://www.bbbaden.ch/"
+    ],
+    "name": "Berufsfachschule BBB",
+    "alpha_two_code": "CH",
+    "state-province": null,
+    "domains": [
+      "stud.bbbaden.ch"
+    ],
+    "country": "Switzerland"
+  },
+  {
+    "web_pages": [
+      "hftm.ch"
```

Comment:
Protocol missing

Reply:
Just noticed it also and fixed it right before